packages/typespec-ts/test/modularIntegration/arrayItemTypes.spec.ts

```diff
@@ -99,6 +115,18 @@ describe("Array Item-Types Client", () => {
           case "nullable-float":
             result = await client.nullableFloatValue.get();
             break;
+          case "nullable-int32":
+            result = await client.nullableInt32Value.get();
+            break;
+          case "nullable-string":
+            result = await client.nullableStringValue.get();
+            break;
+          case "nullable-boolean":
+            result = await client.nullableBooleanValue.get();
+            break;
+          case "nullable-model":
```

Comment:
let's comment this out for now, and left a TODO here https://github.com/Azure/autorest.typescript/issues/2705